next/src/index.ts

```diff
@@ -1 +1,2 @@
 export { createHeadlessForm, CreateHeadlessFormOptions, ValidationOptions } from './form'
+export { modifySchema as modify } from './modify-schema'
```

Comment:
q: Why calling it `modify-schema.ts`? Why not just modify? Or a folder `modify/index`?

Reply:
because we already have a file called `mutations`, for mutating fields. Having a ´modify` and a `mutations` field can be confusing IMO, so I decided to be more specific on what we're modifying. (I kept the export the same though)